tools/teleop_roslibpy.py

```
@@ -275,6 +288,27 @@ def start(self) -> None:
+                # Target speed is computed from inputs; actual output ramps towards it
+                # at BLDC_RAMP_STEP units/tick to prevent mechanical shock on start/stop.
+                # Published every tick (not just on change) to keep the firmware's
+                # 500 ms watchdog fed reliably.
+                bldc_target = 0
+                if deadman_held:
+                    rb_held = self._js.get_button(BLDC_CW_BUTTON)
+                    rt_val = (self._js.get_axis(AXIS_RT) + 1.0) / 2.0  # 0..1
+                    if rb_held:
+                        bldc_target = 80
+                    elif rt_val > 0.05:
+                        bldc_target = -int(round(rt_val * 127))
```

Ruff (0.15.15)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)